src/module/dy/fns/get-item.js

```diff
@@ -1,9 +1,17 @@
 const assert = require('assert');
+const Joi = require('joi-strict');
 
-module.exports = (model, onNotFound_, setDefaults) => async (key, {
-  toReturn = null,
-  onNotFound = onNotFound_
-} = {}) => {
+module.exports = (model, onNotFound_, setDefaults) => async (key, kwargs = {}) => {
+  Joi.assert(key, Joi.object());
+  Joi.assert(kwargs, Joi.object().keys({
+    // eslint-disable-next-line newline-per-chained-call
+    toReturn: Joi.array().items(Joi.string()).unique().min(1).allow(null).optional(),
```

Comment:
I don't think we want to allow null here. It won't break anything. We just default it for a conditional check below.